service/routes.py

```diff
@@ -141,6 +141,34 @@ def delete_recommendations(recommendation_id):
     return "", status.HTTP_204_NO_CONTENT
 
 
+######################################################################
+# UPDATE AN EXISTING PET
+######################################################################
+@app.route("/recommendations/<int:recommendation_id>", methods=["PUT"])
+def update_recommendations(recommendation_id):
+    """
+    Update a Recommendation
+
+    This endpoint will update a Recommendation based the body that is posted
```

Comment:
typo, and "based on the posted body" would be clearer

Reply:
based *on*

Reply:
Fixed.